tools/FakeItEasy.Build/Program.cs

```diff
@@ -51,45 +47,36 @@ public class Program
             }
         };
 
-        private static readonly string OutputDirectory = Path.GetFullPath("artifacts/output");
-
         public static void Main(string[] args)
         {
-            Target("default", DependsOn("unit", "integ", "spec", "approve", "pack"));
-
-            Target("outputDirectory", () => Directory.CreateDirectory(OutputDirectory));
-
-            Target("logsDirectory", () => Directory.CreateDirectory(LogsDirectory));
+            Target("default", dependsOn: TestSuites.Keys.Concat(new[] { "pack" }));

```

Comment:
I was worried about this when I saw it. I was worried that the test execution order would be some crazy thing, and I like the current order (well, I like `approve` being last; I'd almost prefer `spec` first, since we're moving so many tests there). I see they ran in the same orer, and wonder if this was just luck or what. Any idea?

Reply:
Good catch. Apparently, [the order is non-deterministic], so I'll revert this change.